### ISSUE TITLE
rebar3 warning in elixir projects

### ISSUE THREAD

**Issue by sneako** [g-andrade/locus#16]
First off, thanks for the great library! I have tried out several BEAM geoip libraries, and locus is by far my favorite and the most performant one I have found.

I am using locus in an Elixir project, and am constantly getting the nice big warning:
```
        !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
        !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
        *********************************************************************

        [locus] You're **strongly** incentivized to use `rebar3`.
        Compatibility with rebar 2 is unmaintained and will be removed in the near future.

        *********************************************************************
        !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
        !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```
from both locus and tls_certificate_check. 

You probably do not use Elixir yourself, but I was hoping to begin a discussion on how we could modify the check so that locus can be used within Elixir projects without emitting these warnings.

**Comment by g-andrade** [comment]
Oh! It never occurred to me that `mix` could be detected as being `rebar 2`... but it makes sense, given [the way that detection is performed](https://github.com/g-andrade/locus/blob/1.12.0/rebar.config.script#L2).

Perhaps the check could be expanded into looking for `mix`'s presence - I'll need to look for appropriate module and function names (which will likely need to be prefixed with 'Elixir.', assuming they'll be named after Elixir-style upper case atoms.)

**Comment by g-andrade** [comment]
Fixed under [e7bfbcb](https://github.com/g-andrade/locus/commit/e7bfbcb9ef3b21a0be9b96f9ba604f9253a396e6) and released as `1.12.1`.

Closing this; please re-open if for some reason the fix doesn't work. Cheers!

**Comment by sneako** [comment]
It works, thanks!
